doc/whats_new/v0.20.rst

```diff
@@ -188,6 +189,13 @@ Misc
   one was added (:ref:`environment_variable`).
   :issue:`11166`by `Gael Varoquaux`_
 
+Datasets
+
+- Added :func:`datasets.fetch_openml` to fetch any dataset from `OpenML <http://openml.org>`.
+  OpenML is a free, open data sharing platform and will replace mldata, which
+  is no longer maintained. :issue:`9908` by `Andreas Müller`_  and :user:`Jan N. van Rijn
```

Comment:
Maybe,

"""
OpenML is a free, open data sharing platform and will be used instead of mldata as it provides better service availability.
"""

I mean it's used that way in scikit-learn, but the previous formulation seems very general. Unless there is a official statement from mldata that it's no longer maintained and that OpenML should be used instead?

Reply:
OK